active_model_serializers.gemspec

```diff
@@ -57,4 +57,10 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'grape', ['>= 0.13', '< 1.0']
   spec.add_development_dependency 'json_schema'
   spec.add_development_dependency 'rake', ['>= 10.0', '< 12.0']
+
+  spec.post_install_message = <<-EOF
+NOTE: The default key case for the JsonApi adapter has changed to dashed.
+See https://github.com/rails-api/active_model_serializers/blob/master/docs/general/key_transform.md
+for more information on configuring this behavior.
```

Comment:
Mehhhh, the post_install_message is not a good way to issue a deprecation I think. I mean, there is even a gem for ignoring post install messages that is pretty popular.... https://github.com/tpope/gem-shut-the-fuck-up

Why not just issue a real deprecation warning?